redaxo/src/addons/backup/lib/backup.php

```diff
@@ -329,6 +329,10 @@ public static function exportDb($filename, array $tables = null)
             }
         }
         foreach ($tables as $table) {
+            if (!preg_match('/^[a-z0-9._-]+$/i', $table)) {
```

Comment:
Warum überhaupt hier der Check, und nicht direkt in `rex_sql::showCreateTable()`?
Und warum dort dann nicht `$this->escapeIdentifier` nutzen?
Damit sollten wir doch eigentlich dann sicher sein.